### PR TITLE
Feature shibv421

### LINKED ISSUE #53: Upgrade to version 4.2.1
https://shibboleth.atlassian.net/wiki/spaces/IDP4/pages/1265631499/ReleaseNotes#4.2.1-(April-18,-2022)

4.2.1 (April 18, 2022)
This is a fixed package to address a regression that caused the spymemcached library to be omitted. Only users of the memcache storage option would be impacted.

4.2.0 (April 15, 2022)
This is a relatively “light” feature release that mainly exists to support API changes needed for some of our plugins, which have been updated along with it and require this version to use.

### PR DESCRIPTION
**Pull request by trsau**
Upgrade to v4.2.1

Resolves #53

### CODE REVIEW

**Review by ArthurZheng — APPROVED** [review]
👍 nice work!

**Comment by trsau** [comment]
Now resolves #58

**Comment by trsau** [comment]
Some minor changes to align config with what is provided in Shibboleth v4.2.1. The inclusion of the SCHAC schema into attribute is the most significant change.

**Comment by ArthurZheng** [comment]
Awesome work @trsau! I'll make changes accordingly in rapid-idp-ansible and also rapid-idp-manager.

**Review by ArthurZheng — APPROVED** [review]
👍 nice work! 🚢

File: assets/idp.example.edu.dist/idp/conf/idp.properties [hunk 1]
```diff
@@ -187,11 +194,18 @@ idp.bindings.inMetadataOrder = false
 # Whether to require logout requests/responses be signed/authenticated.
 #idp.logout.authenticated = true
 
+# Whether to handle logout lacking response endpoonts as asynchronous.
```
Comment (ArthurZheng) [inline assets/idp.example.edu.dist/idp/conf/idp.properties:197]:
```suggestion
# Whether to handle logout lacking response endpoints as asynchronous.
```

**Pull request merged**